locales/ko_KR/LC_MESSAGES/duckduckgo.po

```diff
@@ -3804,37 +3804,37 @@ msgstr "가장 많이 방문한 사이트 목록 표시"
 
 msgctxt "settings"
 msgid "Show favicons for each result"
-msgstr "모든 검색 결과에 파비콘 표시"
+msgstr "모든 검색 결과에 파비콘을 표시합니다."
 
 msgctxt "settings"
 msgid "Show horizontal lines at result page breaks"
-msgstr "검색 결과 페이지마다 구분선 표시"
+msgstr "검색 결과 페이지마다 가로 구분선을 보여줍니다."
 
 msgctxt "settings"
 msgid "Show links to instructions for how to add DuckDuckGo to your browser"
 msgstr ""
-"사용 중인 브라우저에 DuckDuckGo를 추가하는 방법을 안내하는 메시지 보여주기"
+"사용 중인 브라우저에 DuckDuckGo를 추가하는 방법을 안내하는 메시지를 보여줍니다."
 
 msgctxt "settings"
 msgid "Show most visited links on new tab page"
 msgstr ""
 
 msgctxt "settings"
 msgid "Show occasional reminders to add DuckDuckGo to your browser"
-msgstr "DuckDuckGo 검색 엔진 사용 추천 메시지 보여주기"
+msgstr "DuckDuckGo 검색 엔진 사용 추천 메시지를 보여줍니다."
 
 msgctxt "settings"
 msgid "Show occasional reminders to add DuckDuckGo to your devices"
-msgstr "기기에 DuckDuckGo 설치 미리 알림 표시"
+msgstr "DuckDuckGo 검색 엔진 사용 추천 메시지를 보여줍니다."
```

Comment:
Could you please double check this? The previous version sounds more accurate. When translating it back to English the first one reads "Display a reminder to install DuckDuckGo on your device", while your version reads "Shows recommendations for using the DuckDuckGo search engine.". Of course it's machine translation, so it might be wrong. Thanks!

Reply:
The current translation sounds a bit weird to me but I had no idea how to make this translation better without context. Is this a string for the DuckDuckGo app?

Reply:
The current translation sounds a bit weird to me but I had no idea how to make this translation better without context. Is this a string for the DuckDuckGo app?

Reply:
This setting controls whether we display reminders on the search page to install DDG on other devices. For example, on the web we might prompt you to try out our mobile apps.